app/views/devise/registrations/new.html.erb

```diff
@@ -1,29 +1,27 @@
 <h2>Sign up</h2>
-
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
-
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
```

Comment:
- [ ] (OPTIONAL) Great job adding the name field 👍 It would be great if you could add fields for the other user inputs like bio and photo 💯 
- [ ] (OPTIONAL) Also, it would be great if you could configure your routes to take a user to the signup page if that user hasn't signed up yet 😄